### PR TITLE
Dp24 jbrowse only

### PR DESCRIPTION
**Pull request by DLBPointon**
This Re-adds the lib folder that got deleted?

Implements the JBrowse only subworkflow, I have confirmed this to work locally.

### CODE REVIEW

**Comment by DLBPointon** [comment]
There were a further 2 modules using the no longer available quay.io/sanger-tol/cram-filter... container.

Now corrected.

**Review by weaglesBio — APPROVED** [review]
Only minor text changes

File: modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf
[the remark this answers is not in the document]
Comment (DLBPointon) [reply modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf:1]: Fixed

File: modules/local/fkutils/fkprof/main.nf
[the remark this answers is not in the document]
Comment (DLBPointon) [reply modules/local/fkutils/fkprof/main.nf:1]: fixed

**Pull request merged**